front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -249,50 +249,19 @@ const AdminProjectEventEdit = () => {
     setAttendanceLimitVisible(toggleValue);
     setAttributeDiff({
       ...attributeDiff,
-      maximum_attendees: null,
+      maximum_attendees: toggleValue === false ? null : 100,
     });
   };
 
-  const handleMaximumAttendeesChange = (value: string) => {
+  const handleMaximumAttendeesChange = (maximum_attendees: string) => {
     setSubmitState('enabled');
-    const numberValue = value ? parseInt(value, 10) : null;
-
-    // Create a custom error message that will be used for both client and API validation
-    const maximumAttendeesErrorMessage = {
-      error: formatMessage(messages.maximumAttendeesError),
-    };
-
-    // Check if the new value is less than the current attendees count
-    if (
-      numberValue !== null &&
-      attendeesCount &&
-      numberValue < attendeesCount
-    ) {
-      setErrors({
-        ...errors,
-        maximum_attendees: [maximumAttendeesErrorMessage],
-      });
-    } else {
-      // Clear error if it exists - create a copy of the errors object
-      if (
-        typeof errors === 'object' &&
-        !Array.isArray(errors) &&
-        'maximum_attendees' in errors
-      ) {
-        // Create a new object without the maximum_attendees property
-        const { maximum_attendees: _maximum_attendees, ...restErrors } =
-          errors as {
-            maximum_attendees: CLError[];
-            [key: string]: any;
-          };
-        setErrors(restErrors);
-      }
-    }
-
     setAttributeDiff({
       ...attributeDiff,
-      maximum_attendees: numberValue,
+      // If maximum_attendees is an empty string, set it to 0.
+      // Number('') returns 0.
+      maximum_attendees: Number(maximum_attendees),
```

Comment:
It was odd that we could save an empty "Maximum attendees" field. With this line, deleting a number value will automatically convert the empty string to 0. 

Now, it should not be possible to save maximum attendees as `null`, unless you explicitly toggle off this field.

Reply:
Or does is it simply set the field value to zero, then raise the BE error if the form is saved with this value set?

Reply:
Good questions. It did work as you set the `min` attribute on `Input` to "1". If you try saving with "0", the browser has a built-in error thanks to that `min` attribute, which prevents submission. 

However, if someone were to remove that min attribute from our code for some reason, we would receive an empty error message (BE validation would kick in, but the copy for this BE error was not defined). So I've used the BE error message for this use case as well, and added an error message (and removed the min attribute/browser validation bit, which is also more consistent).